web/src/components/molecules/TopPage/TopPageContents.tsx

```
@@ -1,8 +1,19 @@
 import Loading from "@marketplace/components/atoms/Loading";
 import Pagination from "@marketplace/components/atoms/Pagination";
+import Tabs from "@marketplace/components/atoms/Tabs";
 import PluginsList, { Plugin } from "@marketplace/components/molecules/PluginsList";
 import SearchArea from "@marketplace/components/molecules/SearchArea";
+import { useT } from "@marketplace/i18n";
 import { styled } from "@marketplace/theme";
+import { Version } from "@marketplace/types";
+
+type TabKeys = "0" | "1";
+
+type Tabs = {
```

Biome

[error] Shouldn't redeclare 'Tabs'. Consider to delete it or rename it.

'Tabs' is defined here:


(lint/suspicious/noRedeclare)